asserts/asserts.go

```diff
@@ -63,8 +63,9 @@ var (
 
 // Assertion types without a definite authority set (on the wire and/or self-signed).
 var (
-	SerialProofType   = &AssertionType{"serial-proof", nil, assembleSerialProof, noAuthority}
-	SerialRequestType = &AssertionType{"serial-request", nil, assembleSerialRequest, noAuthority}
+	SerialProofType       = &AssertionType{"serial-proof", nil, assembleSerialProof, noAuthority}
+	SerialRequestType     = &AssertionType{"serial-request", nil, assembleSerialRequest, noAuthority}
+	AccountKeyRequestType = &AssertionType{"account-key-request", nil, assembleAccountKeyRequest, noAuthority}
```

Comment:
It's a curious edge case. We might actually end up with an authority-id on that assertion. It could make sense for a different key to sign that assertion for revoking, for example, and perhaps even for adding if we had the proof of ownership signature inside the assertion itself.

The current case is simple though, so probably a good start.